clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml

```diff
@@ -37,7 +43,133 @@ spec:
     #       node-role.kubernetes.io/worker: ""
     #     ptpEventConfig:
     #       storageType: "storage-class-http-events"
-
+    - fileName: PtpConfigGmWpc.yaml
+      policyName: "config-policy"
+      spec:
+        profile:
+        - name: "grandmaster"
+          phc2sysOpts: -r -u 0 -m -O -37 -N 8 -R 16 -s ens6f0 -n 24
+          ptp4lConf: |
+            [ens6f0]
```

Comment:
Just checking if I got this right. This means that the port acting as master port in  `falcon` and `fiesta` is always named ens6f0? Is there any other cluster in `group-smcx12`?

Reply:
Right, we're creating a group based policy that's aligned to the SKU of systems i have ordered, where cards are in the same slot across.